pkg/skaffold/verify/verify_mux.go

```diff
@@ -52,71 +50,56 @@ func (m VerifierMux) GetVerifiers() []Verifier {
 	return m.verifiers
 }
 
-func (m VerifierMux) GetAccessor() access.Accessor {
-	var accessors access.AccessorMux
-	for _, deployer := range m.verifiers {
-		accessors = append(accessors, deployer.GetAccessor())
-	}
-	return accessors
-}
-
 func (m VerifierMux) GetLogger() log.Logger {
 	var loggers log.LoggerMux
-	for _, deployer := range m.verifiers {
-		loggers = append(loggers, deployer.GetLogger())
+	for _, verifier := range m.verifiers {
+		loggers = append(loggers, verifier.GetLogger())
 	}
 	return loggers
 }
 
 func (m VerifierMux) GetStatusMonitor() status.Monitor {
 	var monitors status.MonitorMux
-	for _, deployer := range m.verifiers {
-		monitors = append(monitors, deployer.GetStatusMonitor())
+	for _, verifier := range m.verifiers {
+		monitors = append(monitors, verifier.GetStatusMonitor())
 	}
 	return monitors
 }
 
-func (m VerifierMux) GetSyncer() sync.Syncer {
-	var syncers sync.SyncerMux
-	for _, deployer := range m.verifiers {
-		syncers = append(syncers, deployer.GetSyncer())
-	}
-	return syncers
-}
-
 func (m VerifierMux) RegisterLocalImages(images []graph.Artifact) {
-	for _, deployer := range m.verifiers {
-		deployer.RegisterLocalImages(images)
+	for _, verifier := range m.verifiers {
+		verifier.RegisterLocalImages(images)
 	}
 }
 
 func (m VerifierMux) Verify(ctx context.Context, w io.Writer, as []graph.Artifact) error {
-	for _, deployer := range m.verifiers {
+	for _, verifier := range m.verifiers {
 		ctx, endTrace := instrumentation.StartTrace(ctx, "Deploy")
 		runHooks := false
-		deployHooks, ok := deployer.(verifierWithHooks)
+		verifyHooks, ok := verifier.(verifierWithHooks)
 		if ok {
-			runHooks = deployHooks.HasRunnableHooks()
+			runHooks = verifyHooks.HasRunnableHooks()
 		}
 		if runHooks {
-			if err := deployHooks.PreDeployHooks(ctx, w); err != nil {
+			if err := verifyHooks.PreDeployHooks(ctx, w); err != nil {
 				return err
 			}
 		}
```

Comment:
Do we want to support hooks for verify in the future? Or is it safe to delete this code?

Reply:
We can had this in the future, doesn't make sense to have unused code here (there is no schema for hooks in verify currently).

Reply:
Removed this for now, doesn't make sense to have unused stub/dead code.  We do want to add this in the future, created https://github.com/GoogleContainerTools/skaffold/issues/8472 to track this